### PR TITLE
Update bluelinky.html

### PR DESCRIPTION
**Pull request by PierreLevres**
Added brand Kia/Hyundai to enable for Bluelinky 7.0.0

### CODE REVIEW

File: bluelinky.html [hunk 1]
```diff
@@ -162,6 +162,7 @@
             region: { value: "EU", required: true },
             pin: { required: true, validate: RED.validators.number() },
             vin: { required: true }
+            brand: { value: "H", required: true }
```
Comment (Hacksore) [inline bluelinky.html:165]: @PierreLevres I believe that `brand` should be either `hyundai` or `kia` now.

**Review by Hacksore — CHANGES_REQUESTED** [review]

File: bluelinky.html [hunk 2]
```diff
@@ -366,4 +367,11 @@
         <label for="node-config-input-vin"><i class="icon-bookmark"></i> Vin</label>
         <input type="text" id="node-config-input-vin">
     </div>
-</script>
\ No newline at end of file
+    <div class="form-row">
+        <label for="node-config-input-brand"><i class="icon-bookmark"></i> Brand</label>
+        <select id="node-config-input-brand">
+            <option value="K">Kia</option>
```
Comment (Hacksore) [inline bluelinky.html:373]: The same goes for the values here as well.

**Review by Hacksore — CHANGES_REQUESTED** [review]

File: bluelinky.html (hunk 1 above)
Comment (PierreLevres) [reply bluelinky.html]: Indeed, I changed it in a later commit

File: bluelinky.html (hunk 2 above)
Comment (PierreLevres) [reply bluelinky.html]: Indeed, I changed it in a later commit

**Pull request merged**